### PR TITLE
Readme template for SIGs 

### PR DESCRIPTION
**Pull request by jparsana**
This is related to Issue #96

### CODE REVIEW

**Comment by jparsana** [comment]
@tedsuo Add PR per our talk yesterday

@bogdandrutu @SergeyKanzhelev @yurishkuro For Context: We should have a sig specific folder for each sig in the community repo. Each folder should contain atleast this a instance of this template that contains info related to the SIG. each SIG (either owner or memebers) would be responsible maintain this doc.  This way if anybody wants more info about the SIG, who is in it, when they meet etc is in one place.

**Comment by jparsana** [comment]
I signed it

File: SIG-Readme-Template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# <Name> Special Interest Group
+<Goals of this SIG>
+
+## Team
```
Comment (SergeyKanzhelev) [inline SIG-Readme-Template.md]: what we found there is not really a team. You can just point to (at)open-telemetry/dotnet-approvers and (at)open-telemetry/dotnet-maintainers. Other people are coming ad-hoc.

File: SIG-Readme-Template.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+* <Member Name> (@Github id), <Company Name>
+* <Member Name> (@Github id), <Company Name>
+* <Member Name> (@Github id), <Company Name>
+
+## Meeting Info
+* Meeting Name
+  * Dial in info
+  * [Meeting Minutes](URL).
```
Comment (SergeyKanzhelev) [inline SIG-Readme-Template.md]: I'd suggest to put it on top

Comment (SergeyKanzhelev) [reply SIG-Readme-Template.md]: this is kind of what people are checking out before deciding whether to join the meeting. Not other way around

File: SIG-Readme-Template.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# <Name> Special Interest Group
```
Comment (SergeyKanzhelev) [inline SIG-Readme-Template.md]: gitter link should be on top as well

**Comment by SergeyKanzhelev** [comment]
It's great to have a template. Can you please try to move .NET, (maybe all of them later) to demonstrate how it will look like.

**Comment by bogdandrutu** [comment]
@jparsana please sign the CLA.

**Comment by bogdandrutu** [comment]
@jparsana can you please sign the CLA and respond to comments.

**Comment by SergeyKanzhelev** [comment]
this PR was inactive for too long. Please re-open if it's still needed

**Pull request closed without merging**